c2rust-ast-exporter/build.rs

```diff
@@ -273,23 +281,62 @@ impl LLVMInfo {
 
         let llvm_config = find_llvm_config();
         let llvm_config_missing = "
+        Couldn't find `llvm-config`. Make sure `llvm-config` is on $PATH then
+        re-build.
+        ";
+        let llvm_config_libdir_missing = "
         Couldn't find LLVM lib dir. Try setting the `LLVM_LIB_DIR` environment
         variable or make sure `llvm-config` is on $PATH then re-build. For example:
 
           $ export LLVM_LIB_DIR=/usr/local/opt/llvm/lib
         ";
+        let llvm_config_cmakedir_missing = "
```

Comment:
Arguably these should be named `llvm_libdir_missing`, and `llvm_cmakedir_missing`.